terra-node-dev/test/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
Should these match/reference the repo Apache license?

Reply:
this isn't a real package